### PR TITLE
NEW: adds file upload info to APIv2

### PR DESCRIPTION
**Pull request by glennr**

### CODE REVIEW

**Comment by glennr** [comment]
[File uploading (1)](https://trello.com/c/2CBwruZB/51-file-uploading-1)

File: guides/file-uploads.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+```
+query Node {
+  node(id: "msg_4023561927") {
+    ... on EmailMessage {
+      attachments {
+        nodes {
+          contentId
```
Comment (jscheel) [inline guides/file-uploads.md:7]: What is contentId?

Comment (glennr) [reply guides/file-uploads.md:7]: field :contentId,
        String,
        null: true,
        description: 'The RFC2387 identifier (if from an email)'

File: guides/file-uploads.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+          downloadUrl
```
Comment (jscheel) [inline guides/file-uploads.md:8]: This should be a signed, expiring url, right?

**Review by jscheel — APPROVED** [review]

**Pull request merged**